tests/simpletest.py

```diff
@@ -156,7 +160,7 @@ def test_to_from_base64(self, filename):
         try:
             os.unlink(filename)
             bf = pybloomfilter.BloomFilter.from_base64(filename, b64,
-                                                       perm=0775)
+                                                       perm=0x775)
```

Comment:
This test is failing:
```
======================================================================
FAIL: test_to_from_base64 (tests.simpletest.SimpleTestCase)
----------------------------------------------------------------------
Traceback (most recent call last):
  File "/Users/felipe/git-repos/pybloomfiltermmap/tests/__init__.py", line 16, in _wrapped
    return method(*args, **kwargs)
  File "/Users/felipe/git-repos/pybloomfiltermmap/tests/simpletest.py", line 165, in test_to_from_base64
    self.assertBfPermissions(bf, '0775')
  File "/Users/felipe/git-repos/pybloomfiltermmap/tests/simpletest.py", line 147, in assertBfPermissions
    self.assert_(oct_mode.endswith(perms),
AssertionError: False is not true : unexpected perms 0o100565

----------------------------------------------------------------------
Ran 25 tests in 161.650s
```
Based on [this answer](https://stackoverflow.com/questions/36745577/how-do-you-create-in-python-a-file-with-permissions-other-users-can-write), I made the following change and now the tests run correctly (both on 2.7 and 3.6):
```suggestion
                                                       perm=0o775)
```

Reply:
+1, 0x775 != 0775